### PR TITLE
feat(ingest/vertica): use 3 part naming

### PR DESCRIPTION
**Pull request by Rajasekhar-Vuppala**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

Below are the summary of this PR:

- Deleted Vertica logo image which is duplicated.
- Updated Vertica SQL alchemy dialect version.
- Updated Vertica.py based on the changes done by datahub team.
- Updated snapshot for integration tests based on the changes.

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "VerticaSource":
         return cls(config, ctx)
 
     def get_workunits_internal(self) -> Iterable[Union[MetadataWorkUnit, SqlWorkUnit]]:
+        yield from super().get_workunits_internal()
         sql_config = self.config
         if logger.isEnabledFor(logging.DEBUG):
             # If debug logging is enabled, we also want to echo each SQL query issued.
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py:139]: this if statement and the one below it can be removed, since they're already in `super().get_workunits_internal()`

File: metadata-ingestion/tests/integration/vertica/docker-compose.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ services:
     platform: linux/amd64
     environment:
       APP_DB_USER: "dbadmin"
-      APP_DB_PASSWORD: "abc123"
```
Comment (hsheth2) [inline metadata-ingestion/tests/integration/vertica/docker-compose.yml:7]: why is this changed? I think it's better for our integration tests to test using a password

File: metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py (hunk 1 above)
Comment (Rajasekhar-Vuppala) [reply metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py]: Done.Thanks harshal for your review. Did the changes according to your comments

File: metadata-ingestion/tests/integration/vertica/docker-compose.yml (hunk 2 above)
Comment (Rajasekhar-Vuppala) [reply metadata-ingestion/tests/integration/vertica/docker-compose.yml]: We updated the docker image password. Anyhow we have added the password back. It should be good now

**Review by hsheth2 — APPROVED** [review]
